meta-mender-demo/conf/layer.conf

```diff
@@ -16,5 +16,8 @@ MENDER_STORAGE_TOTAL_SIZE_MB_DEFAULT ?= "608"
 IMAGE_OVERHEAD_FACTOR = "1.0"
 IMAGE_FEATURES += "splash"
 
+# Special for RPi
+ENABLE_UART = "1"
```

Comment:
This is already present in [meta-mender-raspberrypi-demo](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-raspberrypi-demo/classes/rpi-demo-wifi.bbclass#L6), no need to add it here.

Reply:
Indeed :), I will remove

Reply:
> Nice piece of work @texierp! I have a few change requests, but nothing major. We'll have to verify that RaspberryPi indeed works though.

With pleasure @kacf ! PR updated :)